src/services/order.service.js

```diff
@@ -0,0 +1,13 @@
+const orderDao = require('../models/order.dao');
+
+const postOrder = async(user_id, order_number , status_id , price ,quantity) =>{
```

Comment:
```suggestion
const createOrder = async(user_id, order_number , status_id , price ,quantity) =>{
```
마찬가지로 function name 및 parameter의 snake_case 수정해주세요~